### PR TITLE
Add Quartz.Net scheduler

### LINKED ISSUE #2: Set up the scheduler
The nightwatch is essentially a programmable scheduler that periodically performs some tasks, logs and notifies its users about events.

We need to choose some scheduler that could be configured by nightwatch user.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Close #2.

### CODE REVIEW

**Comment by ForNeVeR** [comment]
@gsomix, wanna review?

File: Nightwatch/Scheduler.fs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+let create() : Async<IScheduler> =
+    async {
+        let factory = StdSchedulerFactory()
+        return! factory.GetScheduler()
+    }
+
+let private configureTask (scheduler : IScheduler) (job : IJobDetail, trigger) : Task =
+    upcast scheduler.ScheduleJob(job, trigger)
+
+let configure (schedule : Schedule) (scheduler: IScheduler) : Async<unit> =
```
Comment (gsomix) [inline Nightwatch/Scheduler.fs]: Please change arguments order in the signature ("what to configure? scheduler").

```fsharp
let configure (scheduler: IScheduler) (schedule : Schedule) : Async<unit>
```

File: Nightwatch/CheckerJob.fs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+namespace Nightwatch
+
+open Quartz
+
+type CheckerJob() =
+    static member CheckFunction = "CheckerFunction"
```
Comment (gsomix) [inline Nightwatch/CheckerJob.fs]: Why `"CheckerFunction"`? Not `"CheckFunction"`?

**Comment by ForNeVeR** [comment]
Thanks for your review; the issues are fixed.

**Pull request merged**